internal/cluster/openshift.go

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package cluster
+
+import (
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+// Create a new openshift client using the in cluster config
+func openshiftClient() (*configv1client.ConfigV1Client, error) {
```

Comment:
let's use OpenShift word

Reply:
would not it be more specific if we use like configClient, clusterConfigClient?

Reply:
I just left it as lowercase because I don't think it really needs to be used by anything else. The only reason it's used is because the kubernetes client-go doesn't have access to clusteroperators

Reply:
I was not clear and meant openShift